critterSpec.yaml

```diff
@@ -1,936 +1,1132 @@
+          description: OK
+      summary: Login
+      tags:
+        - auth
+      parameters: []
+  /users/auth/register:
+    post:
+      operationId: register
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RegisterDetails"
+              type: object
+      responses: {}
+      summary: Register
+      tags:
+        - auth
+      parameters: []
+  /vendors:
     get:
-      summary: Get the deck validity rules object
+      operationId: getVendors
       tags:
-        - decks
+        - market
+      summary: Get vendors
       responses:
-        '200':
-          description: OK
+        "200":
+          description: OK - Successful request with response body
           content:
             application/json:
               schema:
-                type: object
-                $ref: '#/components/schemas/DeckRules'
-      operationId: get-decks-validity
-      x-stoplight:
-        id: xp1900zz8v5rj
-  /users/auth/login:
-    post:
-      summary: Login
+                type: array
+                items:
+                  $ref: "#/components/schemas/Vendor"
+  /vendors/{vendorid}/offers:
+    get:
+      operationId: getAVendorsOffers
       tags:
-        - Auth
+        - market
+      summary: get a vendors offers
+      parameters:
+        - $ref: "#/components/parameters/vendorid"
       responses:
-        '200':
-          description: OK
+        "200":
+          description: OK - Successful request with response body
           content:
             application/json:
               schema:
-                type: object
-                $ref: '#/components/schemas/User'
-      operationId: post-users-auth-login
-      x-stoplight:
-        id: 6fmef8g4isswl
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              $ref: '#/components/schemas/LoginDetails'
-  /users/auth/register:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Offer"
     post:
-      summary: Register
+      operationId: createNewOffer
       tags:
-        - Auth
-      responses: {}
-      operationId: post-users-auth-register
-      x-stoplight:
-        id: o0cadttd8hbw1
+        - market
+      summary: create new offer
+      parameters:
+        - $ref: "#/components/parameters/vendorid"
+      responses:
+        "201":
+          description: Created - Resource successfully created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Offer"
       requestBody:
+        required: true
+        description: ""
         content:
           application/json:
             schema:
-              type: object
-              $ref: '#/components/schemas/RegisterDetails'
-  /users/auth/logout:
+              $ref: "#/components/schemas/Offer"
+  /vendors/{vendorid}/offers/{offerid}:
```

Comment:
Will the API know which user is purchasing?

Reply:
yes, its based on who is signed in